EGO Forum Enhancement.user.js

```diff
@@ -288,6 +319,71 @@ function handleBanReport() {
         addMoveButton(button_group, window.location.href);
 }
 
+/**
+ * Adds "On Hold" templates to the menu and increases the size of the explain box.
+ * @param {HTMLElementEventMap} event 
+ * @returns void
+ */
+function handleOnHold(event) {
+    if(event.target.nodeName != 'DIV' || !event.target.classList.contains('overlay-container'))
```

Comment:
Inconsistent spacing

```suggestion
    if (event.target.nodeName != 'DIV' || !event.target.classList.contains('overlay-container'))
```